README.md

```diff
@@ -34,7 +34,7 @@ Data types by Apache Calcite: https://calcite.apache.org/docs/reference.html#dat
 ## Configuration
 
 - **query**: SQL to run (string)
-- **default_timezone**: If the sql type of a column is `date`/`time`/`datetime` and the embulk type is `string`, column values are formatted int this default_timezone. You can overwrite timezone for each columns using column_options option. (string, default: `UTC`)
+- **default_timezone**: Set JDBC connect 'timeZone' param (string)
```

Comment:
Thank you for your comments. I revised 'default_timezone' option in README.md. db7a4e0a6ce11b2bb9cb75ebc5fcd6c7d420107a

1. This is optional and the default value is 'UTC'. 
2. Users specify one of AvailableIDs of java.util.TimeZone. I put several links on README.md
3. The JDBC connection parameter is passed to Calcite engine. The engine uses it as local timezone. For example, LOCALTIMESTAMP is based on the specified timezone.